nativelib/src/main/scala/scala/scalanative/native/unistd.scala

```diff
@@ -0,0 +1,8 @@
+package scala.scalanative
+package native
+
+@extern
+object unistd {
```

Comment:
On the second thought, it's probably fine to stick to the convention of naming `@extern object`-s with the name of the header, but we still need to move it away from the main public API.